library/ssl_tls13_keys.c

```diff
@@ -322,6 +146,9 @@ int mbedtls_ssl_tls13_hkdf_expand_label(
 {
     unsigned char hkdf_label[ SSL_TLS1_3_KEY_SCHEDULE_MAX_HKDF_LABEL_LEN ];
     size_t hkdf_label_len;
+    psa_status_t status = PSA_SUCCESS;
```

Comment:
`status` should be initialized to an error, as a good practice.

Reply:
Fixed: Initialized status to error.